1-exercises/A-reading-list/script.js

```diff
@@ -1,7 +1,33 @@
 function readingList(books) {
   // Write your code here...
+  let getContent = document.querySelector("#content");
+  let ulElement = document.createElement("ul");
+  getContent.appendChild(ulElement);
+
+  for (const book of books) {
+    let liElement = document.createElement("li");
+    ulElement.appendChild(liElement);
+
+    let pElement = document.createElement("p");
+    pElement.innerText = `${book.title} by ${book.author}`;
+    liElement.appendChild(pElement);
+
+    let imgElement = document.createElement("img");
+    imgElement.src = book.bookCoverImage;
+    liElement.appendChild(imgElement);
+
+    if (books.alreadyRead === true) {
```

Comment:
Excellent - you have used JavaScript to build the DOM model.  Did you notice though that all the books have the same red background.  There is something wrong with this line. Can you see?